### DISCUSSION TITLE
XR Tatemae，Masking e-motions.【Feng Baoying】

### DISCUSSION THREAD

**Discussion by sisyphus1280** [realitydeslab/playshop-summer-2024#10] [Proposal]
Chinese TItle：
vr 建前，情感面具

Team：
Feng Baoying

Research Questions:
VR Tatemae，Masking e-motions.

Keywords:
情绪传递、情绪压抑、和谐社会
Emotional transmission,Emotional repression，harmonious society

Artefact / Media:
视频、基于视觉与听觉的ar过滤系统
Video, vR filtering system based on vision and hearing

Abstract:
在社会中，人们总是被要求理性、友善，"情绪化 "被视为贬义词。 因此，为了构建一个更加和平友好的未来网络环境，一个 VR 世界的插件应运而生。 当检测到情绪达到一定阈值时，该插件就会启动，将过激的表达转化为和谐的表达，从而带来一个更加和平友好的社会。 没有争吵，没有悲伤，只有和平与爱。 然而，我希望这样的压制能够引发反思： 虚假的和平真的能带来幸福吗？ 当真实的情感被强制的平静所掩盖，潜在的情感问题能否得到解决，还是只能被掩盖？
In society, people are always asked to be rational and friendly, and "emotional" is regarded as a derogatory term. So in order to build a more peaceful and friendly future network environment, a plug-in for the VR world was born. This plug-in activates when emotions is detected to reach a certain threshold, transforming excessive expressions into harmonious ones, which may bring a more peaceful and friendly society. No quarrels, no sorrow, only peace and love. However, I hope such suppression can prompts reflection: Does false peace truly bring happiness? When genuine emotions are masked by enforced calmness, can underlying emotional issues be resolved, or do they simply remain hidden?

Visual Reference / Illustrations
https://www.youtube.com/watch?v=e2zyjbH9zzA
https://github.com/realitydeslab/playshop-summer-2024/assets/170873077/e0f0777f-4b24-4408-b7d9-acde2534a965

References:
Experimental research on emotion recognition based on brain-computer interface and brain waves
How to use smartphones for less obtrusive ambulatory mood assessment and mood recognition
Crowd Abnormal Behavior Detection Combining Movement and Emotion Descriptors
多元线性回归与决策树的情绪量化识别算法
基于3D特征融合与轻量化CNN的情绪EEG识别

Your Reading List：
The _Craft_of_Research_Fourth_Edition_Booth_Wayne_C._Colomb_Gregory_G._Williams_etc
Anthony_Dunne_Fiona_Raby_-_Speculative_Everything
social psychology

**Comment by sisyphus1280** [comment]
Tagline:
扼杀 "情感"，创造 "和平与爱"。 忽视真正潜在的情感问题。 建立一个 "完全和平的社会"，我们需要付出怎样的代价？
Killing the “emotional”.Creating the “peace and love” . Ignoring real latent emotional problems. What prices should we need to pay to build a “completely peaceful society”？

**Comment by sisyphus1280** [comment]
![1](https://github.com/realitydeslab/playshop-summer-2024/assets/170873077/3d9b7af0-9f03-465a-9e48-85974f508252)

**Reply by sisyphus1280** [reply]
The boy put on his glasses and saw something that surprised him

**Comment by glitch-kraft** [comment]
https://en.wikipedia.org/wiki/Honne_and_tatemae

**Reply by glitch-kraft** [reply]
https://tokyofamilies.net/2014/09/honne-and-tatemae/

**Reply by glitch-kraft** [reply]
![image](https://github.com/realitydeslab/playshop-summer-2024/assets/46427283/50b4bf58-0296-4d46-bc29-de384cacf413)

**Comment by sisyphus1280** [comment]
A VR Tatemae plugin to curb harmful content by detecting and transforming negative emotions and language into positive expressions using real time AI processing.

**Comment by glitch-kraft** [comment]
Masking e-motions. An XR plugin that curbs harmful content by transforming negative emotions and language into smoothed expressions via real-time AI processing. Inspired by Hon-né and taté-maé japanese etiquette.

**Comment by glitch-kraft** [comment]
XR Tatemae [建前], Masking e-motions

**Comment by glitch-kraft** [comment]
@sisyphus1280 https://www.nakasendoway.com/omote-ura-public-and-private-faces/
